packages/client/src/i18n/it/auth.ts

```diff
@@ -59,4 +59,8 @@ export default {
       "I dati relativi alle tue richieste, prenotazioni, acquisti, vendite e altre attività condotte sul Mercatino del Libro verranno utilizzati per condurre analisi statistiche che miglioreranno il servizio che offriamo alle famiglie. Non temere: per salvaguardare la tua privacy, tutte queste informazioni verranno anonimizzate a norma di legge.",
     deleteAccount: "Elimina account",
   },
+  privacyAndToSRegister:
+    "Cliccando Registrati accetti i <a class='text-accent' href='/tos-privacy/tos-{0}.pdf'>Termini di Servizio</a> e la <a class='text-accent' href='/tos-privacy/privacy-policy.pdf'>Privacy Policy</a> del Mercatino.",
```

Comment:
In order to make sure we are considering all the use cases, we must be a little bit more specific here.

```suggestion
    "Cliccando Registrati o registrandoti tramite un sistema di terze parti accetti i <a class='text-accent' href='/tos-privacy/tos-{0}.pdf'>Termini di Servizio</a> e la <a class='text-accent' href='/tos-privacy/privacy-policy.pdf'>Privacy Policy</a> del Mercatino.",
```

Please apply these changes in the English version too.